docs/installation.md

```diff
@@ -457,7 +457,7 @@ This will mainly affect Debian, Ubuntu, and related distributions, or RHEL where
 
 #### golang
 
-Be careful to double-check that the version of golang is new enough (i.e. `go version`), as of January 2022 version is 1.16.x or higher is required.
+Be careful to double-check that the version of golang is new enough (i.e. `go version`), as of May 2024 version is 1.21.x or higher is required.
```

Comment:
I know most (all?) of this was pre-existing, but if you could change to this while you're in there, I'd appreciate it.
```suggestion
Be careful to double-check that the version of Golang is new enough (i.e., `go version`). As of May 2024, version 1.21.x or higher is required.
```